Src/ILGPU/Backends/OpenCL/CLInstructions.Data.cs

```diff
@@ -238,51 +238,59 @@ partial class CLInstructions
             ">="
         };
 
+        private enum CLUnaryCategory
```

Comment:
Can you add a small comment here why we need this enumeration? (It is absolutely clear but a little comment would be helpful in this case, I guess)